### DISCUSSION TITLE
[node] `process.report.getReport()` doesn't return `string`

### DISCUSSION THREAD

**Discussion by adesso-os** [DefinitelyTyped/DefinitelyTyped#63788] [Issues with a @types package] [answered]
[The documentation, back to v14](https://nodejs.org/docs/latest-v14.x/api/report.html), even has this example:
```ts
const report = process.report.getReport();
console.log(typeof report === 'object'); // true

// Similar to process.report.writeReport() output
console.log(JSON.stringify(report, null, 2));
```

From what I can tell, the returned report was never a `string`. Yet, that is [what the typedefs currently state](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1dcd97e06e5b89b61500bf9907df4b0bc0f81ad6/types/node/process.d.ts).

[a comment by an automated account is not shown]

**Comment by SimonSchick** [comment, marked as answer]
This was correct when it was experimental, it hasn't been updated since https://nodejs.org/docs/latest-v11.x/api/process.html#process_process_report_getreport_err

Feel free to raise a PR.

**Reply by adesso-os** [reply]
> Feel free to raise a PR.

I only came by the wrong typedef while researching a problem with Node. I am not invested in this API and have no interest in resolving this, otherwise you'd already have a PR 😊
